nucliadb_models/src/nucliadb_models/search.py

```diff
@@ -1724,6 +1727,11 @@ class FindRequest(BaseSearchRequest):
     query_entities: SkipJsonSchema[Optional[list[KnowledgeGraphEntity]]] = Field(
         default=None, title="Query entities", description="Entities to use in a knowledge graph search"
     )
+    graph_query: SkipJsonSchema[Optional[GraphPathQuery]] = Field(
```

Comment:
You are not propagating this to the GET endpoints. On purpose? It seems like we are likely to forget to add it later unless we make an explicit note to remember.

Reply:
Yeah, it's weird, but it's what I ended up doing for filter expressions. :shrug: 

Maybe we should deprecate GETs